microsoft-edge/progressive-web-apps-chromium/microsoft-store.md

```diff
@@ -132,6 +132,16 @@ To submit your app to the Store, complete the following actions.
     
 After you complete your submission, your app is reviewed, typically within between 24 and 48 hours.  After you receive approval, your PWA is available in the Microsoft Store.  
 
+### Measuring Store-installed PWA usage
+
+Starting in version 91, Microsoft Edge includes a `Referer` header with the **initial launch** of any PWA that has been installed from the Microsoft Store. 
```

Comment:
This is my edit pass.  I used guidance from the Microsoft Writing Style Guide (to avoid 'enables' or 'allows') and Acrolinx (to shorten sentences).
```suggestion
### Measure usage of your Store-installed PWA

When your PWA is initially launched, if the PWA was installed from the Microsoft Store, Microsoft Edge includes the following `Referer` header with the request for the first navigation of your web app.
```
Initial launch, or first navigation?  Might be able to remove start or end of sentence.

Reply:
93?

Reply:
> 93?

It’s been backported to 91 from it’s initial 93-focused release.

> Initial launch, or first navigation?

What you have is fine. Mainly it’s important that we note that it only comes that one time.